### ISSUE TITLE
aws_glue_connection_invalid_connection_type missing support BIGQUERY and other values.

### ISSUE THREAD

**Issue by guitarrapc** [terraform-linters/tflint-ruleset-aws#757]
## tl;dr;

There are `Error: "BIGQUERY" is an invalid value as connection_type (aws_glue_connection_invalid_connection_type)` even when using terraform-aws-provider v5.56.0 or higher.

## Description of issue

terraform-provider-aws v5.56.0 introduces support for `aws_glue_connection`, adding `AZURECOSMOS`, `AZURESQL`, `BIGQUERY`, `OPENSEARCH`, and `SNOWFLAKE` as valid values for the connection_type argument.

> see: https://github.com/hashicorp/terraform-provider-aws/releases/tag/v5.60.0

However tflint still detects the following error.

```
1 issue(s) found:

Error: "BIGQUERY" is an invalid value as connection_type (aws_glue_connection_invalid_connection_type)

  on main.tf line 19:
  19:   connection_type = "BIGQUERY"
```

It seems that [AwsGlueConnectionInvalidConnectionTypeRule.NewAwsGlueConnectionInvalidConnectionTypeRule](https://github.com/terraform-linters/tflint-ruleset-aws/blob/4ee7ea998cd205afcf23e41f935eacfc2d5596ce/rules/models/aws_glue_connection_invalid_connection_type.go#L23) is missing the definition for `BIGQUERY` and possibly other values. Could it be that we need to add these values to the `connection_type` enum? Is this understanding correct?

## Reproduce code

```terraform
resource "aws_glue_connection" "bigquery_connection" {
  name            = "bigquery-connection"
  connection_type = "BIGQUERY"
  connection_properties = {
    SparkProperties = jsonencode({
      secretId = "<reducted>"
    })
  }
}
```

## Versions

* Terraform: v1.9.7
* terraform-provider-aws: 5.77.0
* tflint: 0.54.0
* tflint-ruleset-aws: 0.35.0
